src/scripts/common.ts

```diff
@@ -365,14 +365,41 @@ namespace Scripts {
             Orion.OpenContainer('openContainer');
         }
 
-        static turboRess() {
-            var serialGhost = Orion.FindType('1', '-1', 'ground', "human|fast|dead",1)
-            if (!serialGhost.length) {
+        static turboRess(bandageAfterRess = false) {
+            const closestGhosts = Orion.FindType(-1, -1, 'ground', 'human|dead|fast', 1);
+            if (closestGhosts?.length < 1) {
                 Scripts.Utils.playerPrint('Nevidis zadneho ducha k oziveni');
             }
-            else {
-                Orion.WaitTargetObject(serialGhost[0]);
+            
+            Orion.WaitTargetObject(closestGhosts[0]);
+            Orion.UseType(gameObject.uncategorized.bandy.graphic);
+            Orion.Wait(100);
+            if (bandageAfterRess) {
+                Orion.WaitTargetObject(closestGhosts[0]);
+                Orion.UseType(gameObject.uncategorized.bandy.graphic);
+            }
+        }
+
+        static turboRessFull() {
+            const closestGhosts = Orion.FindType(-1, -1, 'ground', 'human|dead|fast', 1);
+            if (closestGhosts?.length < 1) {
+                Scripts.Utils.playerPrint('Nevidis zadneho ducha k oziveni');
+            }
+            
+            if (Orion.Count(gameObject.uncategorized.krvavaBanda1.graphic) >= 30) {
```

Comment:
Const tricetBand = count(banda1) > 29 ? Banda1 : count(banda2) > 29 ? Banda2 : undefined

Odstran duplikovany kod v podminkach.

Pridej kdyztak parametr na ress obyc bandou kdyz neni dost krvavych (default muze byt true)

Reply:
Nepoznám človeka, čo by mal rád vnorené ternary podmienky :D Trošku som to prerobil, tak uvidíš, či sa ti to dobre číta alebo nie.

Reply:
tak jednou vnorenej ternar muzu :-)

je mi to celkem jedno, ja klera nehraju, tak jsem si jen predstavoval ze by se s nekym domlouval ze ho hodi do full a nakonec by to tak vlastne nedopadlo, ale na to asi srat :)